cob_twist_controller/scripts/test/test_careobot_st_jla_ca_selfcollision_arm_left_direct_head.py

```diff
@@ -141,17 +142,15 @@ def init_pos():
             # save data
             for data_kraken in data_krakens:
                 data_kraken.writeAllData()
-        except rospy.ROSInterruptException as e:
```

Comment:
Why is this bit removed?

Reply:
ROSInterruptException is a child of KeyboardInterrupt...there was an error message about bad except clause order...

Reply:
OK